flink-table/flink-table-common/src/main/java/org/apache/flink/table/plan/stats/ColumnStats.java

```diff
@@ -50,28 +50,72 @@
 	private final Integer maxLen;
 
 	/**
-	 * max value of column values.
+	 * Deprecated because not well supported comparable type,
+	 * e.g. {@link java.util.Date}, {@link java.sql.Timestamp}.
 	 */
-	private final Number max;
+	@Deprecated
+	private final Number maxValue;
 
 	/**
-	 * min value of column values.
+	 * max value of column values, null if the value is not comparable.
```

Comment:
`null if the value is not comparable`? 
I thought null means we don't have such information.